packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -485,7 +485,10 @@ export const getLinksSupplier =
       if (href) {
         href = locationUtil.assureBaseUrl(href.replace(/\n/g, ''));
         href = replaceVariables(href, dataLinkScopedVars, VariableFormatID.UriEncode);
-        href = locationUtil.processUrl(href);
+
+        if (href?.length > 0) {
```

Comment:
nit: would `if (href?.length)` work?

Reply:
Yes but I prefer the more explicit check. Ill change it if someone finds something else.